pom.xml

```diff
@@ -47,10 +47,11 @@
     </issueManagement>
 
     <properties>
-        <jenkins.version>2.60.3</jenkins.version>
+        <jenkins.version>2.164.3</jenkins.version>
         <release.skipTests>false</release.skipTests>
         <maven.javadoc.skip>true</maven.javadoc.skip>
         <findbugs-maven-plugin.version>3.0.4</findbugs-maven-plugin.version>
+        <hamcrest.version>2.2</hamcrest.version>
         <concurrency>1</concurrency>
         <java.level>8</java.level>
         <workflow.version>1.14.2</workflow.version>
```

Comment:
Would be great to switch to https://github.com/jenkinsci/bom in a later pull request. This version of Pipeline is ancient, and I here we are not testing with versions being really used with the target Jenkins core

Reply:
Agree bom is a good move, but that is a separate change.